### PR TITLE
Add a link for instant flow analytics

### PR DESCRIPTION
**Pull request by isuruf**
For easy access

image

cc @lukeolson

### CODE REVIEW

File: course/templates/course/course-base.html [hunk 1]
```diff
@@ -183,13 +183,23 @@
...
       {% else %}
         {% trans "There are some interactive activities going on right now." %}
         {% for i, req in instant_flow_requests %}
           <a class="btn btn-primary"
             href="{% url "relate-view_start_flow" course.identifier req.flow_id %}"
             role="button"><b>{% trans "Go to activity" %} {{ i }} &raquo;</b></a>
+          {% if pperm.view_analytics %}
+          <a class="btn btn-primary"
+            href="{% url "relate-flow_analytics" course.identifier req.flow_id %}"
+            role="button"><b>{% trans "Go to activity" %} {{ i }} {% trans "analytics" %} &raquo;</b></a>
```
Comment (inducer) [inline course/templates/course/course-base.html]: Could you change the verb to "View activity analytics" (in both cases)?

**Comment by isuruf** [comment]
> Do you have a sense where the big drop in code coverage comes from? I have no idea what might be causing this.

In master 3 jobs in Travis reported coverage, but in the PR, only 2 did. See https://codecov.io/gh/inducer/relate/commit/5ae9dbbecca0120bd5ed38fdd8f359706545f58c/build vs https://codecov.io/gh/inducer/relate/commit/4c886cb6b559024124daac79d3a2cf0fdbe6ccf6/build

**Comment by isuruf** [comment]
2nd job in the PR failed due to a network error. https://travis-ci.org/inducer/relate/jobs/578596632#L768

**Comment by inducer** [comment]
Thanks!

**Pull request merged**